KappaAnalysis/interface/KappaSettings.h

```diff
@@ -102,6 +102,8 @@ class KappaSettings: public SettingsBase {
 
 
 	IMPL_SETTING(float, CrossSection);
+    IMPL_SETTING_DEFAULT(float, CrossSectionUp, 0.0)
+    IMPL_SETTING_DEFAULT(float, CrossSectionDown, 0.0)
```

Comment:
I would recommend to use default values, which are not within the scope of the numerical reachable. Better use something negative, e.g. `-1`.

Reply:
Ok, I'll change it to -1, but I think the check > 0 should stay, as the cross section should be positive.